include/libopencm3/stm32/g4/rcc.h

```diff
@@ -192,27 +234,30 @@
 #define RCC_PLLCFGR_PLLQ_SHIFT		21
 #define RCC_PLLCFGR_PLLQ_MASK		0x3
 
-#define RCC_PLLCFGR_PLLQEN		BIT20
+#define RCC_PLLCFGR_PLLQEN		(1 << 20)
 
 /* Division for PLLSAI3CLK, 0 == 7, 1 == 17 */
-#define RCC_PLLCFGR_PLLP		BIT17
+
 #define RCC_PLLCFGR_PLLP_DIV7		0
-#define RCC_PLLCFGR_PLLP_DIV17		RCC_PLLCFGR_PLLP
-#define RCC_PLLPEN			(1 << 16)
+#define RCC_PLLCFGR_PLLP_DIV17		1
+#define RCC_PLLCFGR_PLLP_SHIFT		17
+#define RCC_PLLCFGR_PLLP_MASK		0x1
```

Comment:
we generally don't use mask/shift definitions for single bit fields, is there any particular reason this had to be donne?

Reply:
I think this was done to make the code look more consistent, but you're right; there's insufficient reason to change what was already there. I've fixed now.